src/acceptance/administrator/components/com_users/UserNotesCest.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * @package     Joomla.Tests
+ * @subpackage  Acceptance.tests
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace administrator\components\com_users;
+use Page\Acceptance\Administrator\UserNotesListPage as UserNotesList;
+use Page\Acceptance\Administrator\UserNotesFormPage as UserNotesForm;
+/**
+ * Administrator UserNotes Tests
+ *
+ * @category  Users
+ * @package   Administratorcomponentscom_Users
+ * @author    Samarth sharma <samarthsharma351@gmail.com>
+ * @copyright 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license   Joomla 2005-2018
+ * @link      User Notes
+ * @since     4.0.0
+ */
+class userNotesCest
```

Comment:
Class names must be CamelCase

Reply:
@Samgithub1997  why are you not looking here?